apps/base-docs/src/components/Hero/index.tsx

```diff
@@ -15,10 +20,20 @@ export default function Hero() {
   if (!isReady) {
     return <HeroLoadingState />;
   }
-  if (userVariant === 'treatment') {
-    return <TreatmentHero />;
+
+  logEvent(
+    'exposure__build-onchain-for-less-hero-2024-07-23',
```

Comment:
Hm, longer term lets figure out how to do this automatically in the exposure provider if possible

Reply:
Makes sense. Let's discuss. 

We might be able to use the event that's sent in the `exposureTrackingProvider`.